GameWorld/HexGrid/HexGrid.cs

```diff
@@ -0,0 +1,58 @@
+﻿using HexLib;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Diagnostics;
+
+namespace MonoGameWorld.HexGrid
```

Comment:
Please rename namespace to represent functionality without tool's name(maybe project's name)